3561/1/file_reader.rb

```diff
@@ -0,0 +1,13 @@
+require 'yaml'
+require 'io/console'
+
+#:reek:UtilityFunction
```

Comment:
You can't disable any rubocop or reek errors, please fix all such errors in all files where you disabled it